prow/gerrit/adapter/adapter.go

```diff
@@ -248,18 +289,26 @@ func (c *Controller) Sync() {
 
 		now := time.Now()
 		defer func() {
-			gerritMetrics.changeProcessDuration.WithLabelValues(instance).Observe(float64(time.Since(now).Seconds()))
+			gerritMetrics.changeSyncDuration.WithLabelValues(instance, project).Observe(float64(time.Since(now).Seconds()))
 		}()
 
 		// Ignore the error. It is already logged.
-		changes, _ := c.gc.QueryChangesForProject(instance, project, syncTime, c.config().Gerrit.RateLimit)
 		timeQueryChangesForProject := time.Now()
-		log.WithFields(logrus.Fields{"host": instance, "repo": project, "changes": len(changes), "duration(s)": time.Since(now).Seconds()}).Info("Time taken querying for gerrit changes")
+
+		changes, err := c.gc.QueryChangesForProject(instance, project, syncTime, c.config().Gerrit.RateLimit)
+		queryResult := func() string {
+			if err == nil {
+				return "SUCCESS"
```

Comment:
nit: We could use the constants here too.

Reply:
Will update.